ibm/service/scc/data_source_ibm_scc_posture_get_credential.go

```diff
@@ -0,0 +1,400 @@
+// Copyright IBM Corp. 2022 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package scc
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/scc-go-sdk/v3/posturemanagementv2"
+)
+
+func DataSourceIBMSccPostureGetCredential() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIBMSccPostureGetCredentialRead,
+
+		Schema: map[string]*schema.Schema{
+			"id": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The id for the given API.",
+			},
+			"enabled": &schema.Schema{
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Credentials status enabled/disbaled.",
+			},
+			"type": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Credentials type.",
+			},
+			"name": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Credentials name.",
+			},
+			"description": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Credentials description.",
+			},
+			"display_fields": &schema.Schema{
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "Details the fields on the credential. This will change as per credential type selected.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"ibm_api_key": &schema.Schema{
```

Comment:
Add sensitive : true for all those arguments which have sensitive information like (apikey, secret, password)